phpstan.neon

```diff
@@ -423,3 +423,70 @@ parameters:
+                 - packages/coding-standard/src/Fixer/ArrayNotation/ArrayOpenerAndCloserNewlineFixer.php #28
+                 - packages/coding-standard/src/Fixer/ArrayNotation/StandaloneLineInMultilineArrayFixer.php #47
+                 - packages/coding-standard/src/Fixer/Commenting/ParamReturnAndVarTagMalformsFixer.php #62
+                 - packages/coding-standard/src/Fixer/Commenting/RemoveCommentedCodeFixer.php #59
+                 - packages/coding-standard/src/Fixer/Commenting/RemoveUselessDefaultCommentFixer.php #37
+                 - packages/coding-standard/src/Fixer/Commenting/RemoveUselessDefaultCommentFixer.php #42
+                 - packages/coding-standard/src/Fixer/LineLength/LineLengthFixer.php #85
+                 - packages/coding-standard/src/Fixer/Naming/StandardizeHereNowDocKeywordFixer.php #52
+                 - packages/coding-standard/src/Fixer/Spacing/MethodChainingNewlineFixer.php #53
+                 - packages/coding-standard/src/Fixer/Spacing/SpaceAfterCommaHereNowDocFixer.php #28
+                 - packages/coding-standard/src/Fixer/Strict/BlankLineAfterStrictTypesFixer.php #57
+                 - packages/config-transformer/src/Configuration/Configuration.php #71
+                 - packages/config-transformer/src/Configuration/Configuration.php #76
+                 - packages/console-package-builder/tests/HttpKernel/ConsolePackageBuilderKernel.php #34
+                 - packages/easy-coding-standard/src/DependencyInjection/Extension/EasyCodingStandardExtension.php #14
+                 - packages/easy-coding-standard/src/Error/ErrorAndDiffCollector.php #93
+                 - packages/easy-coding-standard/src/Error/ErrorAndDiffCollector.php #101
+                 - packages/easy-coding-standard/src/Error/ErrorAndDiffCollector.php #127
+                 - packages/easy-coding-standard/src/ValueObject/Error/CodingStandardError.php #44
+                 - packages/easy-testing/src/ValueObject/InputFileInfoAndExpected.php #35
+                 - packages/easy-testing/src/ValueObject/InputFileInfoAndExpected.php #48
+                 - packages/easy-testing/src/ValueObject/InputFileInfoAndExpectedFileInfo.php #37
+                 - packages/latte-to-twig-converter/src/HttpKernel/LatteToTwigConverterKernel.php #12
+                 - packages/monorepo-builder/src/ValueObject/Package.php #43
+                 - packages/php-config-printer/src/DependencyInjection/Extension/PhpConfigPrinterExtension.php #14
+                 - packages/psr4-switcher/src/ValueObject/Psr4NamespaceToPath.php #33
+                 - packages/rule-doc-generator/src/ValueObject/RuleDefinition.php #46
+                 - packages/set-config-resolver/src/ValueObject/Set.php #27
+                 - packages/static-detector/src/ValueObject/StaticClassMethod.php #33
+                 - packages/static-detector/src/ValueObject/StaticClassMethod.php #38
```

Comment:
Single-line stmst like these: https://github.com/symplify/symplify/blob/c2037c9dcc4725e632aded857b841b05cc55f473/packages/static-detector/src/ValueObject/StaticClassMethod.php#L40,
should be skipped.

Many value object only contain `return $x;`, and it is not duplicated logic.

Reply:
implemented.

Reply:
@TomasVotruba how about without `return` statement, something like this:

1. count as 1 statement inside `Expression` statement? 

```php
    public function registerContainerConfiguration(LoaderInterface $loader): void
    {
        $loader->load(__DIR__ . '/../../config/config.php');
    }
```

2. count as 1 statement inside `If_` statement? 

```php
    public function foo()
    {
            if ($foo) {
            
            }
    }
```

Reply:
@TomasVotruba I've added handling 1 expression statement, like single assignment or single method call .

Reply:
To count stmt in `ClassMethod`, this is good enough:

```php
$stmtCount = count((array) $classMethod->stmts)
```

Reply:
that's already used.

```php
        $stmts = $node->stmts;
        if (count($stmts) === 1 && ($stmts[0] instanceof Return_ || $stmts[0] instanceof Expression)) {
            return [];
        }
```

need to save to variable to verify first `$stmts` and ensure there is at least 1 stmt before check stmts[0] instance of Return_ or `Expression` statement.

Reply:
Sorry, I wrote it incorretly.

Anything with 1 stmts and less can be excluded, whatever it is:

```php
$stmtCount = count((array) $classMethod->stmts)

if ($stmtCount <= 1) {
    return [];
}
```

No extra check for `Return_` or `Expression` is needed

Reply:
implemented.